.github/workflows/maven-app-release-with-docs.yml

```diff
@@ -260,14 +260,17 @@ jobs:
       - name: Maven package
         id: build
         run: |
-          mvn clean package -B -f pom.xml -Dserver.username=$NEXUS_USERNAME -Dserver.password=$NEXUS_PASSWORD -Dserver.repo.url=$NEXUS_MAVEN_REPO
           artifactId=$(xmlstarlet pyx pom.xml | grep -v ^A | xmlstarlet p2x | xmlstarlet sel -t -v "/project/artifactId")
           version=$(xmlstarlet pyx pom.xml | grep -v ^A | xmlstarlet p2x | xmlstarlet sel -t -v "/project/version")
-          artifact="${artifactId}-${version}.jar"
-          echo "version=$(echo $version)" >> "${GITHUB_OUTPUT}"
-          echo "artifact=$(echo $artifact)" >> "${GITHUB_OUTPUT}"
-          echo "artifact-id=$(echo $artifactId)" >> "${GITHUB_OUTPUT}"
-          echo "artifact-path=$(echo target/$artifact)" >> "${GITHUB_OUTPUT}"
+          artifact="${artifactId}-${version}"
+          mvn clean package -B -f pom.xml -P release -Dgpg.key.id=$GPG_KEY_ID -Dgpg.key.password=$GPG_KEY_PASSWORD -Dserver.username=$NEXUS_USERNAME -Dserver.password=$NEXUS_PASSWORD -Dserver.repo.url=$NEXUS_MAVEN_REPO
+          mkdir .artifact-$artifact
+          cp target/* -t .artifact-$artifact || :
```

Comment:
nit: with these `|| :` - I'd probably not do it for `cp` and let it fail if there's an issue, and for `rm` you can always use `-f` which doesn't seem to return error. They look more natural commands and less confusing.

Reply:
we'll leave it as is, if it bothers me too much I'll look how to do it differently